citi-onboarding/PubliEasy
# Feature/metrics

## What I did

- Adjusted flexbox of metrics
- Changes images
   - At first i was doing without the litle purple background, then i had to change it;
- I tried to change the font-family but it didn't work that well


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout <branch name>`
   1. `git pull origin <branch name>`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 4h30min

**Resolves:** #<issue number>
